install/kubernetes/helm/istio/charts/pilot/templates/crds.yaml

```diff
@@ -65,11 +65,11 @@ spec:
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: destinationrules.config.istio.io
+  name: destinationrules.networking.istio.io
```

Comment:
need change version v1alpha2 -> v1alpha3 to align with proto message package path.